libraries/AP_Arming/AP_Arming.h

```diff
@@ -49,6 +49,14 @@ class AP_Arming {
     // get bitmask of enabled checks
     uint16_t get_enabled_checks();
 
+    // returns true if a particular check is enabled
+    bool check_enabled(const enum AP_Arming::ArmingChecks check) const;
+    // returns a mavlink severity which should be used if a specific check fails
+    MAV_SEVERITY check_severity(const enum AP_Arming::ArmingChecks check) const;
+    // handle the case where a check fails
+    void check_failed(const enum AP_Arming::ArmingChecks check, bool report, const char *fmt, ...);
```

Comment:
At the moment these aren't used anywhere else but Arming, but I'm looking at the heli parameter checks with some trepidation.